ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -35,6 +39,9 @@ class UpdateLedger blk => ApplyTx blk where
   -- Should be cheap as this will be called often.
   computeGenTxId :: GenTx blk -> GenTxId blk
 
+  -- | Given a 'GenTx', compute its post-serialization size in bytes.
+  computeGenTxSize :: GenTx blk -> TxSizeInBytes
```

Comment:
I think the "compute" prefix in the name is misleading if this is cheap, which it is.

Reply:
Good point, I'll rename it to `getGenTxSize`.

Reply:
Update: we settled on `txSize` instead.

Reply:
Sounds good